File: inspector-interface/src/main/java/org/wartremover/InspectParam.java
package org.wartremover;

import java.net.URL;

public interface InspectParam {
  String[] tastyFiles();
  String[] dependenciesClasspath();
  URL[] wartClasspath();
  String[] errorWarts();
  String[] warningWarts();
  boolean failIfWartLoadError();
}

File: inspector-interface/src/main/scala/org/wartremover/InspectParam.scala
package org.wartremover

final case class InspectParam(
  tastyFiles: List[String],
  dependenciesClasspath: List[String],
  wartClasspath: List[String],
  errorWarts: List[String],
  warningWarts: List[String],
  failIfWartLoadError: Boolean,
)
